models/bamf_nnunet_ct_lungnodules/config/default.yml

```diff
@@ -19,11 +19,14 @@ modules:
     meta: 
       mod: '%Modality'
 
+  NiftiConverter:
+    engine: dcm2niix
+
   NNUnetRunnerV2:
     in_data: nifti:mod=ct
 
   DsegConverter:
-    model_name: bamf_ct_lung_nodule
+    model_name: bamf_nnunet_ct_lungnodules
```

Comment:
Note, this will be displayed in the reader, so if you want to change it to something more human readable (e.g., BAMF CT Lunglobe Detection) you can @vanossj .